pkg/loader/compose/v3.go

```diff
@@ -294,6 +294,20 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 			serviceConfig.Replicas = int(*composeServiceConfig.Deploy.Replicas)
 		}
 
+		// placement:
+		placement := make(map[string]string)
+		for _, j := range composeServiceConfig.Deploy.Placement.Constraints {
+			p := strings.Split(j, " == ")
```

Comment:
missed this, but it's a pain in the butt, but can you test for:
```
      placement:
        constraints: [node.role == manager]
```

as well, as per: https://docs.docker.com/compose/compose-file/#replicas

Reply:
@cdrage , added functional test for the same example and it's working